README.md

```diff
@@ -28,8 +28,27 @@ var MyComponent = onClickOutside(React.createClass({
 }));
 
 ```
+Note that if you try to wrap a React component class without `handleClickOutside(evt)` handler like this, the HOC will throw an error.
+If you want to execute method different than `handleClickOutside(evt)`, you can specify it as a second parameter to `onClickOutside` like this:
 
-Note that if you try to wrap a React component class without `handleClickOutside(evt)` handler, the HOC will throw an error. If you want onClickOutside functionality, you *must* have this function defined.
+```javascript
+// load the HOC:
+var onClickOutside = require('react-onclickoutside');
+
+// create a new component, wrapped by this onclickoutside HOC:
+var MyComponent = onClickOutside(React.createClass({
+  ...,
+  myClickOutsideHandler: function(evt) {
+    // ...handling code goes here...
+  },
+  ...
+}), function (instance) {
```

Comment:
looking at this some more, this should probably simply be part of an options object instead, to be more in line with virtually all modern JS libraries.
```
var Thing = onClickOutside(component, {
  handleClickOutside: component.functionNameForThat
});
```